shell/browser/ui/inspectable_web_contents.h

```diff
@@ -210,6 +210,7 @@ class InspectableWebContents
   bool can_dock_;
   std::string dock_state_;
   bool activate_ = true;
+  bool is_docked_;
```

Comment:
set a default?

Reply:
@nornagon I just copied from what chrome does:
https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/devtools/devtools_window.h;l=417?q=is_docked_;&ss=chromium

Reply:
Chrome sets a default: https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/devtools/devtools_window.cc;l=977;drc=929e790ad17a54008b3dac86cbe30751c968c2ce

Reply:
ah I missed that